java/code/src/com/redhat/rhn/frontend/strings/jsp/StringResource_en_US.xml

```diff
@@ -3384,7 +3384,7 @@ button below, and &lt;b&gt;will be unable to log back in&lt;/b&gt;.</source>
         </context-group>
       </trans-unit>
       <trans-unit id="errata.publish.nochannelsselected">
-        <source>A published patch must belong to at least one channel.</source>
+        <source>At least one channel must be selected for publishing.</source>
```

Comment:
Should we still use the "publish" word in the UI? to me it gives some sense of "an errata can be in another state than 'published'"

Reply:
Reworded.